text-generation-inference/docker/entrypoint.sh

```diff
@@ -5,10 +5,10 @@
 ulimit -l 68719476736
 
 # Hugging Face Hub related
-if [[ -z "${BATCH_SIZE}" ]]; then
-  BATCH_SIZE=2
+if [[ -z "${MAX_BATCH_SIZE}" ]]; then
+  MAX_BATCH_SIZE=4
```

Comment:
you could have done the change from 2 to 4 on the next commit, but anyway that's fine.

Reply:
Yes i though about that but thanks for pointing it out